doc/netplan.md

```diff
@@ -556,7 +564,7 @@ These options are available for all types of interfaces.
 
      ``type`` (scalar)
      :    The type of route. Valid options are "unicast" (default),
-          "unreachable", "blackhole" or "prohibit".
+          "unreachable", "blackhole",  "prohibit" or "wireguard".
```

Comment:
So a wireguard interface may have both 'unicast' routes that specify actual routing tables; and 'wireguard' routes that map to peers?

This seems rather confusing to me; I'm inclined to take back my earlier suggestion here, in favor of using a separate 'peer' notation.

Reply:
Done. I switched back to the original `peers:` schema.